Mage.Sets/src/mage/cards/g/GrabThePrize.java

```diff
@@ -0,0 +1,59 @@
+package mage.cards.g;
+
+import java.util.Collection;
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.condition.Condition;
+import mage.abilities.costs.common.DiscardCardCost;
+import mage.abilities.decorator.ConditionalOneShotEffect;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.effects.common.DamagePlayersEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.TargetController;
+import mage.game.Game;
+import mage.util.CardUtil;
+
+/**
+ * @author Jamaninja
+ */
+public final class GrabThePrize extends CardImpl {
+
+    public GrabThePrize(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{1}{R}");
+
+        // As an additional cost to cast this spell, discard a card.
+        this.getSpellAbility().addCost(new DiscardCardCost());
+
+        // Draw two cards.
+        this.getSpellAbility().addEffect(new DrawCardSourceControllerEffect(2));
+
+        this.getSpellAbility().addEffect(new ConditionalOneShotEffect(
```

Comment:
Miss comment with full ability text

Reply:
Sorry, this was my first card. Should I include the comment in a commit the next time I upload a card?

Reply:
You can edit and add commit to your version, it will be available in that PR too.